basics/editor-setup.md

```diff
@@ -79,7 +89,7 @@ echoes it back, but if you make a file that contains nothing but a
 ***
 
 If you have trouble with this tutorial, please
-[tell me about it](../contact-me.md) and I'll make this tutorial better,
+[tell me about it,](../contact-me.md) and I'll make this tutorial better,
```

Comment:
If you want to fix this, please do it in a separate pull request. The right thing to change would be the `update-ends.py` script. 

```suggestion
[tell me about it](../contact-me.md) and I'll make this tutorial better,
```